twoliter/embedded/Makefile.toml

```diff
@@ -354,11 +354,29 @@ dependencies = ["setup"]
 script_runner = "bash"
 script = [
 '''
+declare -a CARGO_MANIFESTS
+declare -a CARGO_DIGESTS
+mkdir -p .cargo/vendor
+if [ ! -f ".cargo/config.toml" ]; then
+    touch .cargo/config.toml
+fi
```

Comment:
This doesn't seem necessary - `cargo vendor` will create these directories and doesn't complain if the config file is missing.

Reply:
this is because of the marker file

Reply:
removed